integration-tests/validator-services-tests.postman_collection.json

```diff
@@ -1562,7 +1562,7 @@
             "type": "text/javascript",
             "exec": [
               "const uuid = require('uuid');",
-              "if (!environment.sessionId) {",
+              "if (!pm.environment.sessionId) {",
```

Comment:
@simon-20 I was making these changes in Postman. I can't find this one. Do you know where it is?

Reply:
@BibianaC, it is a pre-request script in the "Ad Hoc Validate - Error Handling" folder. You can work it out by looking at the JSON, following the trail up the tree: the code at line 1565 is an event, and if you scroll up in the JSON file you can see which folder it belongs to.